.github/workflows/pull-request.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
     steps:
       - name: Build version
         run: echo ${{ env.VERSION }}
+        env: 
+          VERSION: ${GITHUB_RUN_ID}
```

Comment:
this looks redundant. it's declared in L7 and because it isn't a secret we don't need to make it local. although it wouldn't harm, being the version number, we probably want to keep it global.

Reply:
Thanks !